framework/PageDialogs/BulkConfirmationDialog.tsx

```diff
@@ -31,6 +31,10 @@ const ConfirmBoxDiv = styled.div`
   height: 64px;
   display: flex;
   align-items: center;
+
+  &:focus {
+    outline: none;
```

Comment:
@jamestalton any suggestions here? I used that to avoid to show the focus on the div itself

Reply:
As per discussion with @jamestalton that the current behavior seems to be the proper one based on https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/checkbox_role#keyboard_interactions Closing this PR.